### PR TITLE
Sctp default sndinfo

### PR DESCRIPTION
**Pull request by hoelscher**
Add support for [gs]etsockopt syscall for option SCTP_DEFAULT_SNDINFO

### CODE REVIEW

File: gtests/net/packetdrill/symbols_linux.c [hunk 1]
```diff
@@ -107,7 +107,10 @@ struct int_symbol platform_symbols_table[] = {
 	{ SCTP_MAXSEG,                      "SCTP_MAXSEG"                     },
 	{ SCTP_DELAYED_SACK,                "SCTP_DELAYED_SACK"               },
 	{ SCTP_MAX_BURST,                   "SCTP_MAX_BURST"                  },
+	{ SCTP_DEFAULT_SNDINFO,             "SCTP_DEFAULT_SNDINFO"            },
+#if 0
 	{ SCTP_EVENT,                       "SCTP_EVENT"                      },
+#endif
 	{ SCTP_PEER_ADDR_PARAMS,            "SCTP_PEER_ADDR_PARAMS"           },
```
Comment (tuexen) [inline gtests/net/packetdrill/symbols_linux.c]: What about the SCTP_UNORDERED, SCTP_ADDR_OVER, SCTP_ABORT, SCTP_EOF, SCTP_SENDALL symbols?
Can you add them, please?

File: gtests/net/packetdrill/symbols_linux.c [hunk 2]
```diff
@@ -107,7 +107,10 @@ struct int_symbol platform_symbols_table[] = {
 	{ SCTP_MAXSEG,                      "SCTP_MAXSEG"                     },
 	{ SCTP_DELAYED_SACK,                "SCTP_DELAYED_SACK"               },
 	{ SCTP_MAX_BURST,                   "SCTP_MAX_BURST"                  },
+	{ SCTP_DEFAULT_SNDINFO,             "SCTP_DEFAULT_SNDINFO"            },
```
Comment (tuexen) [inline gtests/net/packetdrill/symbols_linux.c]: Not sure if Linux supports it yet. Will commit this and test...

File: gtests/net/packetdrill/symbols_linux.c (hunk 1 above)
Comment (hoelscher) [inline gtests/net/packetdrill/symbols_linux.c]: For Linux that are in an enum.

File: gtests/net/packetdrill/symbols_linux.c (hunk 2 above)
Comment (hoelscher) [inline gtests/net/packetdrill/symbols_linux.c]: In my Linux it is a define

File: gtests/net/packetdrill/symbols_linux.c (hunk 1 above)
Comment (tuexen) [inline gtests/net/packetdrill/symbols_linux.c]: So: Can you add them?

File: gtests/net/packetdrill/symbols_linux.c (hunk 2 above)
Comment (tuexen) [inline gtests/net/packetdrill/symbols_linux.c]: Great.

**Pull request merged**